assets/js/vue-apps/past-grants.js

```diff
@@ -232,7 +268,7 @@ function init(STORAGE_KEY) {
                 this.trackUi('Sort results', newSort);
             },
 
-            updateResults(urlPath, queryString = null) {
+            updateResults(urlPath, currentQueryString = null) {
```

Comment:
This is the change here, renaming `queryString` to `currentQueryString` to avoid shadowing the query string module above. The rest is prettier autoformatting.